public_api/public_api_basics.ipynb

```diff
@@ -0,0 +1,531 @@
+{
```

Comment:
<p>why does it access it as <code>df['id'][3]</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SystemInternal/SystemCookbook/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>